README.md

```diff
@@ -2,6 +2,24 @@
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/RedHatInsights/insights-operator-ldapauth)](https://goreportcard.com/report/github.com/RedHatInsights/insights-operator-ldapauth)[![Build Status](https://travis-ci.org/RedHatInsights/insights-operator-ldapauth.svg?branch=master)](https://travis-ci.org/RedHatInsights/insights-operator-ldapauth)
 
+## Overview
+
+The whole system consists of several components: CLI client, LDAP Auth service, and the Insights operator instrumentation service:
+
+[component]: doc/components.png "Components"
```

Comment:
Maybe it will be better to add it like image? `![Components](doc/components.png)`